instructions/docs/add-the-encryption-sdk.md

```diff
@@ -0,0 +1,230 @@
+# Exercise 1: Add the AWS Encryption SDK
+
+## Background
+
+Right now, the Document Bucket supports storing files (or documents, or other blobs of data) in S3, and indexing them in DynamoDB. This allows Document Bucket users to share files with other users, or store them for retrieval later. The DynamoDB entries provide fast lookups to the content of the bucket, along with metadata context for each bucket item.
```

Comment:
Why is this the project we've chosen? What concepts am I learning?

Reply:
Should we include a note about configuring buckets to be publicly accessible?

Reply:
Yes. Got both